client/src/app/site/base/base-filter.service/base-filter-list.service.ts

```diff
@@ -382,6 +382,32 @@ export abstract class BaseFilterListService<V extends BaseViewModel> implements
      */
     protected abstract getFilterDefinitions(): OsFilter<V>[];
 
+    private async loadFilters(): Promise<OsFilter<V>[] | null> {
+        return this.parseFilters(await this.activeFiltersStore.load<V>(this.storageKey));
+    }
+
+    private parseFilters(oldFilters: OsFilter<V>[]): OsFilter<V>[] {
+        const newFilterDefs = this.getFilterDefinitions();
+        newFilterDefs.forEach(definition => {
+            const oldDefinition = oldFilters.filter(old => old.property === definition.property);
```

Comment:
`find` would be better suited here, since you ignore all except the first item anyway